pom.xml

```diff
@@ -96,6 +96,7 @@
         <docker-java.version>3.2.7</docker-java.version>
         <kafka.version>3.0.0</kafka.version>
         <slf4j.version>1.6.1</slf4j.version>
+        <gson.version>2.8.5</gson.version>
```

Comment:
Why GSON and what is it used for? I think all of the Strimzi projects use Jackson.